app/facades/movie_facade.rb

```diff
@@ -1,53 +1,95 @@
 class MovieFacade
 
-    def self.top_movies_endpoint(page_num)
-        sort_by = 'vote_average.desc'
-        conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}&vote_count.gte=300")
-    end
+# def self.top_movies_endpoint(page_num)
```

Comment:
This code has been moved out to the service class, and will be deleted after final review